cdhweb/pages/management/commands/exodus.py

```diff
@@ -262,12 +335,16 @@ def image_exodus(self):
         # delete all images and collections prior to run
         # (clear out past migration attempts)
         Image.objects.all().delete()
-        Collection.objects.exclude(pk=get_root_collection_id()).delete()
+        # Collection.objects.exclude(pk=get_root_collection_id()).delete()
+        # fix collection tree so root collection is up to date
+        # FIXME: this is still not working; seems to error on alternate runs
+        # Collection.fix_tree()
```

Comment:
Still an issue? Do we want to do anything about this?

If it is still a problem, I'm inclined to not delete collections. They shouldn't change once the logic is correct.

Reply:
Still a problem and agreed. Let's just not worry about deleting them